src/esp/bindings/SimBindings.cpp

```diff
@@ -430,9 +430,21 @@ void initSimBindings(py::module& m) {
       .def("set_environment_keyframe",
            &AbstractReplayRenderer::setEnvironmentKeyframe,
            R"(Set the keyframe for a specific environment.)")
-      .def_static("environment_grid_size",
-                  &AbstractReplayRenderer::environmentGridSize,
-                  R"(Dimensions of the environment grid.)");
+      .def_static(
+          "environment_grid_size", &AbstractReplayRenderer::environmentGridSize,
+          R"(Get the dimensions (tile counts) of the environment grid.)")
+      .def(
```

Comment:
a property is maybe more pythonic?

Reply:
`AbstractReplayRenderer::environmentGridSize` takes the environment count as a parameter and calculates the environment grid.

The API could be improved by returning the actual internal state. This goes beyond the scope of this changeset however.